lang/cs/Org.Apache.REEF.Experimental.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Org.Apache.REEF.Experimental.Tests")]
```

Comment:
I modeled my AssemblyInfo.cs after other project as
`
using System.Reflection;
using System.Runtime.InteropServices;

[assembly: AssemblyTitle("Org.Apache.REEF.Experimental")]
[assembly: AssemblyDescription("Experimental Library for REEF")]
[assembly: AssemblyProduct("Org.Apache.REEF.Experimental")]

[assembly: Guid("c8ab57a8-98c0-4e6e-a99e-c5b62c2b6997")]
`

And now it failed to build and said
`
error CS0234: The type or namespace name 'Tang' does not exist in the namespace 'Org.Apache.REEF' (are you missing an assembly reference?) 
error CS0234: The type or namespace name 'Utilities' does not exist in the namespace 'Org.Apache.REEF' (are you missing an assembly reference?) 
`

I have Org.Apache.REEF.Tang and Utilities in my References. Any ideas?